.ci/redmine_matrix_job.yaml

```diff
@@ -0,0 +1,59 @@
+---
+job: LibVMA-redmine-issue
+
+registry_host: harbor.mellanox.com
+registry_auth: swx-storage
+
+kubernetes:
+  privileged: true
+  cloud: swx-k8s-spray
+  nodeSelector: 'beta.kubernetes.io/os=linux'
+
+  limits: '{memory: 1Gi, cpu: 1000m}'
+  requests: '{memory: 1Gi, cpu: 1000m}'
+
+failFast: false
+timeout_minutes: 20
+env:
+  MAIL_FROM: jenkins@nvidia.com
+
+volumes:
+  - {mountPath: /hpc/local/inst/hpc-internal-tools, hostPath: /hpc/local/inst/hpc-internal-tools}
+
+runs_on_dockers:
+# ubuntu 16.04 is used due to script dependency on python2 old print format
+  - {name: 'ub16.04-base', url: 'harbor.mellanox.com/swx-infra/x86_64/ubuntu16.04/base', arch: 'x86_64'}
+
+steps:
+  - name: Redmine
+    containerSelector:
+      - "{name: 'ub16.04-base', variant: 1}"
+    run: |
+      #!/bin/bash -eExl
+      env
+      sleep 300
```

Comment:
did you check why we need this?

Reply:
this does not seem to matter at all in the script context, will remove this